src/theory/bv/theory_bv_rewriter.cpp

```diff
@@ -97,6 +97,45 @@ Node TheoryBVRewriter::rewriteViaRule(ProofRewriteRule id, const Node& n)
   return Node::null();
 }
 
+TrustNode TheoryBVRewriter::expandDefinition(Node node)
+{
+  Node expanded = eliminateOverflows(node);
+  return TrustNode::mkTrustRewrite(node, expanded, nullptr);
+}
+
+Node TheoryBVRewriter::eliminateOverflows(Node node)
+{
+  Node res = node;
+  if (true)
```

Comment:
Redundant.

Reply:
oops. Fixed in: 08675920742da7f285c458dfbdf83b972bcf65a6